src/components/Accordion/Accordion.tsx

```diff
@@ -55,7 +55,7 @@ class Accordion extends AutoControlledComponent<Extendable<IAccordionProps>, any
     /** Primary content. */
     children: PropTypes.node,
 
-    /** Additional classes. */
+    /** Additional css class name or space separated class names to apply **/
```

Comment:
this might be simplified a bit, as `className` is React concept, so the `space-separated` aspects could be omitted from the description:

`Additional css class name(s) to apply`